content/howtos/setup-with-HA.md

```diff
@@ -9,7 +8,14 @@ Home Assistant is an open-source home automation platform that allows users to c
 
 Currently, Home Assistant supports conversation-based agents and services. As of writing this, OpenAIs API is supported as a conversation agent; however, access to your homes devices and entities is possible through custom components. Local based services, such as LocalAI, are also available as a drop-in replacement for OpenAI services.
 
-An example custom integration to utilize Local-based services in Home-LLM. Home-LLM is a Home Assistant integration developed by Alex O'Connell ([acon96](https://github.com/acon96)) that allows for a completely local Large Language Model acting as a personal assistant. Using LocalAI as the backend is one of the supported platforms .
+## There are multiple custom integrations available:
+
+Please note that both of the projects are similar in term of visual interfaces, they seem to be derived from the official Home Assistant plugin: [OpenAI Conversation](https://www.home-assistant.io/integrations/openai_conversation/) (to be confirmed)
```

Comment:
I would delete this line as is, without details about its similarties.  There are actually 2 other custom components not mentioned here.

Reply:
As it is a open documentation. If there is two other components, that should be added here. Someone else will surely complete the information.